src/Mailman.Server/ClientApp/src/components/create-merge/Date.js

```diff
@@ -0,0 +1,67 @@
+export function getDateString() {
+  const year = new Date().getFullYear().toString()
+  const month = new Date()
+    .getMonth()
+    .toString()
+    .padStart(2, '0')
+  const day = new Date()
+    .getDate()
+    .toString()
+    .padStart(2, '0')
+  const hour = new Date()
+    .getHours()
+    .toString()
+    .padStart(2, '0')
+  const minute = new Date()
+    .getMinutes()
+    .toString()
+    .padStart(2, '0')
+  const second = new Date()
+    .getSeconds()
+    .toString()
+    .padStart(2, '0')
+  return (
+    year + '-' + month + '-' + day + ' ' + hour + ':' + minute + ':' + second
+  )
+}
+
+export function getUtcDateString() {
```

Comment:
Is there an easier way to do this?
(i.e. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString)?